src/pages/get-eth.tsx

```diff
@@ -170,39 +170,40 @@ const GetETHPage = ({ data }: PageProps<Queries.GetEthPageQuery>) => {
           >
             <Translation id="page-get-eth-where-to-buy-desc" />
           </Text>
-          <Text
-            fontSize="xl"
-            lineHeight="140%"
-            maxWidth="45ch"
-            color="text200"
-            textAlign="center"
-            mb={8}
-          >
-            <Translation id="page-get-eth-where-to-buy-desc-2" />
-          </Text>
+          <br />
           <Box as={EthPriceCard} mb={8} />
-          <ButtonLink
-            to="#country-picker"
-            onClick={() =>
-              trackCustomEvent({
-                eventCategory: "Search by country button",
-                eventAction: "click",
-                eventName: "search_by_country",
-              })
-            }
-          >
-            <Translation id="page-get-eth-search-by-country" />
-          </ButtonLink>
```

Comment:
im not aware of having an agreement on that

Reply:
I know we had discussed this at one point to avoid pushing people straight to centralized exchanges from the hero. But sounds like we're not totally in agreement at this moment, so to help prevent that from holding up the rest of the PR, I can revert this and bring in the rest.